dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/deletefiles/DeleteFilesPopup.java

```diff
@@ -5,22 +5,32 @@
 import com.spectralogic.dsbrowser.gui.components.ds3panel.ds3treetable.Ds3TreeTableValue;
 import com.spectralogic.dsbrowser.gui.util.Ds3Task;
 import com.spectralogic.dsbrowser.gui.util.Popup;
+import javafx.collections.FXCollections;
 import javafx.collections.ObservableList;
 import javafx.scene.control.TreeItem;
 
 public final class DeleteFilesPopup {
     public static void show(final Ds3Task deleteTask, final Ds3PanelPresenter ds3PanelPresenter, final Ds3TreeTablePresenter ds3TreeTablePresenter) {
         final DeleteFilesView deleteView = new DeleteFilesView(deleteTask, ds3TreeTablePresenter, ds3PanelPresenter);
         if (ds3PanelPresenter != null) {
-            final ObservableList<TreeItem<Ds3TreeTableValue>> selectedPanelItems = ds3PanelPresenter.getDs3TreeTableView().getSelectionModel().getSelectedItems();
+            ObservableList<TreeItem<Ds3TreeTableValue>> selectedPanelItems = ds3PanelPresenter.getDs3TreeTableView().getSelectionModel().getSelectedItems();
+            if (null == selectedPanelItems || selectedPanelItems.size() == 0) {
+                selectedPanelItems = FXCollections.observableArrayList();
+                selectedPanelItems.add(ds3PanelPresenter.getDs3TreeTableView().getRoot());
+            }
             changeLabelText(selectedPanelItems, deleteView);
         } else if (ds3TreeTablePresenter != null) {
-            final ObservableList<TreeItem<Ds3TreeTableValue>> selectedMenuItems = ds3TreeTablePresenter.ds3TreeTable.getSelectionModel().getSelectedItems();
+             ObservableList<TreeItem<Ds3TreeTableValue>> selectedMenuItems = ds3TreeTablePresenter.ds3TreeTable.getSelectionModel().getSelectedItems();
+            if (null == selectedMenuItems || selectedMenuItems.size() == 0) {
```

Comment:
I believe we have some `Guard.isNullOrEmpty` methods which can be used to perform this check